packages/0install/0install.2.12/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "talex5@gmail.com"
+authors: "zero-install-devel@lists.sourceforge.net"
+homepage: "http://0install.net/"
+bug-reports: "https://github.com/0install/0install/issues"
+dev-repo: "https://github.com/0install/0install.git"
+build: [
+  [make]
+]
+depends: [
+  "yojson"
+  "xmlm"
+  "ounit"
```

Comment:
It could be. In this version it runs the tests whenever ounit is installed (and skips them otherwise).
I've made a proper fix upstream now to add a build-test action:
https://github.com/0install/0install/commit/6fb9813534dfe568e9b91ad2fd2f7268368b4ca2